src/civic/index.js

```diff
@@ -462,100 +496,78 @@ const processEvidenceRecord = async (opt) => {
+const downloadEvidenceRecords = async (url, trustedCurators) => {
+    const records = [];
+    const errorList = [];
     const counts = {
         error: 0, exists: 0, skip: 0, success: 0,
     };
-    let expectedPages = 1,
-        currentPage = 1;
-
-    const allRecords = [];
-    const errorList = [];
-
-    // get the aproved entries
-    while (currentPage <= expectedPages) {
-        const url = `${urlTemplate}&page=${currentPage}`;
-        logger.info(`loading: ${url}`);
-        const resp = await request({
-            json: true,
-            method: 'GET',
-            uri: url,
-        });
-        expectedPages = resp._meta.total_pages;
-        logger.info(`loaded ${resp.records.length} records`);
-        allRecords.push(...resp.records);
-        currentPage++;
-    }
-
-    // now find entries curated by trusted curators
-    const trustedSubmissions = [];
-
-    // NOTE: purposefully not async and in a for-loop to avoid spamming their API with requests
-    for (const submitter of Array.from(new Set(trustedCurators))) {
-        const { results } = await request({
-            body: {
-                entity: 'evidence_items',
-                operator: 'AND',
-                queries: [{ condition: { name: 'is_equal_to', parameters: [`${submitter}`] }, field: 'submitter_id' }],
-                save: true,
-            },
-            json: true,
-            method: 'POST',
-            uri: `${baseUrl}/evidence_items/search`,
-        });
-
-        trustedSubmissions.push(...results);
-    }
 
-    let submitted = 0;
+    const evidenceItems = [];
+    const query = fs.readFileSync(path.join(__dirname, 'evidenceItems.graphql')).toString();
 
-    for (const record of trustedSubmissions) {
-        if (record.status === 'submitted') {
-            submitted += 1;
-            allRecords.push(record);
+    // Get accepted evidenceItems
+    logger.info(`loading accepted evidenceItems from ${url}`);
+    const accepted = await requestEvidenceItems(url, {
+        query,
+        variables: {
+            status: 'ACCEPTED',
+        },
+    });
+    logger.info(`fetched ${accepted.length} accepted entries from CIViC`);
+    evidenceItems.push(...accepted);
+
+    // Get submitted evidenceItems from trusted curators
+    for (const curator of Array.from(new Set(trustedCurators))) {
+        if (!Number.isNaN(curator)) {
+            logger.info(`loading submitted evidenceItems by trusted curator ${curator} from ${url}`);
+            const submittedByATrustedCurator = await requestEvidenceItems(url, {
+                query,
+                variables: {
+                    status: 'SUBMITTED',
+                    userId: parseInt(curator, 10),
+                },
+            });
+            evidenceItems.push(...submittedByATrustedCurator);
         }
     }
-    logger.info(`loaded ${submitted} unaccepted entries from trusted submitters`);
+    const nbUnaccepted = evidenceItems.length - accepted.length;
```

Comment:
minor: nb isn't a abbreviation i've seen for number before. I would use something more common like num? or even count? `submittedCount` `numSubmitted` etc

Reply:
It's my french that shines through, lol. submittedCount it is now.